charts/newrelic-k8s-metrics-adapter/templates/secret.yaml

```
@@ -8,3 +10,5 @@ metadata:
 type: Opaque
 stringData:
   personalAPIKey: {{ .Values.personalAPIKey | required "personalAPIKey must be set" | quote }}
+{{- end }}
+{{- end}}
```

Missing End of line. Also you can use the helper here too:
```suggestion
  {{ include "newrelic-k8s-metrics-adapter.customSecretPersonalApiKeyKey" . }}: {{ .Values.personalAPIKey | required "personalAPIKey must be set" | quote }}
{{- end }}
{{- end}}

```